src/services/feed_validator.py

```
@@ -42,38 +43,44 @@ async def validate_feed(self, url: str) -> Tuple[bool, Optional[str]]:
         for attempt in range(2):
             try:
-                async with aiohttp.ClientSession(timeout=self.timeout, headers=self.headers) as session:
+                async with aiohttp.ClientSession(
+                    timeout=self.timeout, headers=self.headers
+                ) as session:
                     async with session.get(url, allow_redirects=True) as response:
                         if response.status != 200:
                             if attempt == 0:  # Only log first attempt failures
-                                logging.warning(f"First attempt failed for {url}: HTTP {response.status}")
+                                logging.warning(
+                                    f"First attempt failed for {url}: HTTP {response.status}"
```

Ruff (0.8.0)

Logging statement uses f-string

(G004)